alphalens/tests/test_performance.py

```diff
@@ -835,70 +772,96 @@ def test_factor_rank_autocorrelation(self,
-                             [0.0, 0.242861]]),
-                           (6, 6, True, False,
-                            [[0.075, 0.0], [0.075, 0.0], [0.075, 0.0],
-                             [0.075, 0.0], [0.075, 0.0], [0.075, 0.0],
-                             [0.075, 0.0], [0.075, 0.0], [0.075, 0.0],
-                             [0.075, 0.0], [0.075, 0.0], [0.075, 0.0],
-                             [0.075, 0.0]]),
-                           (6, 6, True, True,
-                            [[0., 0.], [0., 0.], [0., 0.], [0., 0.], [0., 0.],
-                             [0., 0.], [0., 0.], [0., 0.], [0., 0.], [0., 0.],
-                             [0., 0.], [0., 0.], [0., 0.]]),
-                           ])
-    def test_common_start_returns(self, before, after, mean_by_date, demeaned,
+    @parameterized.expand([
+        (
+            2, 3, False, False,
+            [[4.29107e-4, 0.42568354], [-0.02281746, 0.21643620],
+             [0.00000000, 0.00000000], [0.075000000, 0.24186773],
+             [0.21250000, 0.53190963], [0.428250000, 0.89993039]],
+        ),
+        (
+            3, 2, False, True,
+            [[0.0, 0.64345251], [0.0, 0.42568354],
+             [0.0, 0.21643620], [0.0, 0.00000000],
+             [0.0, 0.24186773], [0.0, 0.53190963]],
+        ),
+        (
+            3, 5, True, False,
+            [[0.06700058, 0.0], [4.29107e-4, 0.0], [-0.02281746, 0.0],
+             [0.00000000, 0.0], [0.07500000, 0.0], [0.212500000, 0.0],
+             [0.42825000, 0.0], [0.74522500, 0.0], [1.196182500, 0.0]],
+        ),
+        (
+            1, 4, True, True,
+            [[0., 0.], [0., 0.], [0., 0.],
+             [0., 0.], [0., 0.], [0., 0.]],
+        ),
+        (
+            6, 6, False, False,
+            [[0.54102049, 1.50387253], [0.33326965, 1.16584103],
+             [0.17703136, 0.88447159], [0.06700058, 0.64345251],
+             [4.29107e-4, 0.42568354], [-0.0228175, 0.21643620],
+             [0.00000000, 0.00000000], [0.07500000, 0.24186773],
+             [0.21250000, 0.53190963], [0.42825000, 0.89993039],
+             [0.74522500, 1.38620088], [1.19618250, 2.04972912],
+             [1.82727625, 2.96841637]],
+        ),
+        (
+            6, 6, False, True,
+            [[0.0, 1.50387253], [0.0, 1.16584103], [0.0, 0.88447159],
+             [0.0, 0.64345251], [0.0, 0.42568354], [0.0, 0.21643620],
+             [0.0, 0.00000000], [0.0, 0.24186773], [0.0, 0.53190963],
+             [0.0, 0.89993039], [0.0, 1.38620088], [0.0, 2.04972912],
+             [0.0, 2.96841637]],
+        ),
+        (
+            6, 6, True, False,
+            [[0.5410205, 0.0], [0.33326965, 0.0], [0.1770314, 0.0],
+             [0.0670006, 0.0], [4.29107e-4, 0.0], [-0.0228175, 0.0],
+             [0.0000000, 0.0], [0.07500000, 0.0], [0.2125000, 0.0],
+             [0.4282500, 0.0], [0.74522500, 0.0], [1.1961825, 0.0],
+             [1.8272763, 0.0]],
```

Comment:
These (and the other values above) changed based on https://github.com/quantopian/alphalens/pull/361/files#diff-39e40ed256ddc7619de160850cb6f6a2L848-R689. @jmccorriston and I believe it should in fact change, but these numbers could use a look.